src/dune_rules/melange/melange.ml

```diff
@@ -5,6 +5,8 @@ module Module_system = struct
     | Es6
     | CommonJs
 
+  let default = (CommonJs, ".js")
```

Comment:
Should default be es6? cc @davesnx

Reply:
I'd rather be conservative here and default to CommonJS, which also aligns with the melange defaults.

Reply:
For now, I agree we should default to cjs, since ES6 module is getting stable across the ecosystem.